carta/cpp/core/ScriptedClient/ScriptFacade.cpp

```diff
@@ -742,15 +742,19 @@ QStringList ScriptFacade::centerImage( const QString& controlId ) {
 }
 
 QStringList ScriptFacade::getCenterPixel( const QString& controlId ) {
-    QStringList resultList("");
+    QStringList resultList;
     Carta::State::CartaObject* obj = _getObject( controlId );
     if ( obj != nullptr ){
         Carta::Data::Controller* controller = dynamic_cast<Carta::Data::Controller*>(obj);
         if ( controller != nullptr ){
-            resultList = controller->getCenterPixel();
-            if ( resultList[0] == "null" ) {
+            QPointF center = controller->getCenterPixel();
+            if ( center.x() == -0.0 && center.y() == -0.0 ) {
```

Comment:
That actually sounds good to me. I'll see if I can get that to work. The negative 0 solution was something I came up with after reading this in the QPointF documentation:

> Since this function treats +0.0 and -0.0 differently, points with zero-valued coordinates where either or both values have a negative sign are not defined to be null points.

-0.0 seems to work, although I admit it does look strange.